src/components/books-cover/books-cover.html

```diff
@@ -0,0 +1,13 @@
+<a class="back-button" ui-sref="books" target="Self">Atras</a>
+<div class="book-cover">
+  <img class="badge" ng-src="{{$ctrl.badge}}" alt="badge">
+  <img class="image-cover" ng-src="{{ $ctrl.book.image_url ? $ctrl.book.image_url : $ctrl.bookCover }}" alt="Book cover">
```

Comment:
Since it's either the default cover or the image's, I think the ternary isn't really needed. You can use the `or` operator (`||`) like this:

```suggestion
  <img class="image-cover" ng-src="{{ $ctrl.book.image_url || $ctrl.bookCover }}" alt="Book cover">
```